### PR TITLE
Feature/swipable modal

### PR DESCRIPTION
**Pull request by Akash-Kait**
Added swipeable modal feature

### CODE REVIEW

**Comment by vinitraj10** [comment]
@Akash-Kait  please update some screenshot/video of the component you have made

File: .eslintrc.js [hunk 1]
```diff
@@ -37,6 +37,12 @@ module.exports = {
     '@typescript-eslint/no-unused-vars': 'error',
     // 'object-curly-newline': ['error', { multiline: true }],
     'object-curly-newline': 'off',
+    'prettier/prettier': [
+      'error',
```
Comment (vinitraj10) [inline .eslintrc.js:41]: why this config added??

**Review by vinitraj10 — CHANGES_REQUESTED** [review]
@Akash-Kait please create documentation too for this component,please check how other component docs has been written

File: .eslintrc.js (hunk 1 above)
Comment (Akash-Kait) [reply .eslintrc.js:41]: end of line error for windows.

**Comment by Akash-Kait** [comment]
> @Akash-Kait please create documentation too for this component,please check how other component docs has been written

Done

**Comment by Akash-Kait** [comment]
> @Akash-Kait please update some screenshot/video of the component you have made

Done

**Pull request closed without merging**